locales/en-US/translation.json

```diff
@@ -0,0 +1,164 @@
+  "Like!": "Like!",
+  "Viewer": "Viewer",
+  "Cancel": "Cancel",
+  "Create": "Create",
+  "Admin": "Admin",
+  "New": "New",
+
+  "Update": "Update",
+  "Update Page": "Update Page",
+  "Warning": "Warning",
+
+  "Sign in": "Sign in",
+  "Sign up is here": "Sign up",
+  "Sign in is here": "Sign in",
+  "Registration": "Registration",
+  "Register by Google Account": "Register by Google Account",
+  "Sign in by Google Account": "Sign in by Google Account",
+  "Register with this Google Account": "Register with this Google Account",
+  "Example": "Example",
+  "Taro Yamada": "Taro Yamada",
+
+  "List View": "List",
+  "Timeline View": "Timeline",
+  "Presentation Mode": "Presentation Mode",
+
+  "Created": "Created",
+  "Last updated": "Updated",
+
+  "Share": "Share",
+  "Share Link": "Share Link",
+  "Markdown Link": "Markdown Link",
+
+  "Unportalize": "Unportalize",
+
+  "View this version": "View this version",
+  "View diff": "View diff",
+
+  "User ID": "User ID",
+  "User Settings": "User Settings",
+  "User Infomation": "User Infomation",
+  "User Basic Infomation": "User Basic Infomation",
+  "Name": "Name",
+  "Email": "Email",
+  "Language": "Language",
+  "English": "English",
+  "Japanese": "Japanese",
+  "Profile Image Setting": "Profile Image Setting",
+  "Current Image": "Current Image",
+  "Delete Image": "Delete Image",
+  "Are you sure to delete?": "Are you sure to delete?",
+  "Updated": "Updated",
+  "Upload new image": "Upload new image",
+  "Google Setting": "Google Setting",
+  "Connected": "Connected",
+  "Disconnect": "Disconnect",
+
+  "Create today's": "Create today's ...",
+  "Memo": "memo",
+  "Input page name": "Input page name",
+  "Input page name empty ok": "Input page name (empty ok)",
+  "New Page": "New Page",
+  "Create under": "Create page under: <code>%s</code>",
+
+  "Table of Contents": "Table of Contents",
+
+  "Public": "Public",
+  "Only who know the link": "Only who know the link",
+  "Specified users": "Specified users",
+  "Only me": "Only me",
+  "Share link of this page": "Share link of this page",
+
+  "Show latest": "Show latest",
+  "Load latest": "Load latest",
+  "was edited this page": "was edited this page.",
+
+  "Deleted Pages": "Deleted Pages",
+  "Logout": "Logout",
+
+  "page_register": {
+    "notice": {
+      "restricted": "New registration to this wiki is restricted.",
+      "restricted_defail": "In order to start using it, approval by the administrator is necessary after new registration.",
+      "google_account_continue": "Please decide user ID, name and password and continue registration."
+    },
+    "form_help": {
+      "email": "In this wiki, only the following e-mail address can be registered.",
+      "password": "Please enter at least 6 letters, numbers, alphanumeric characters, symbols etc for password.",
+      "user_id": "The user ID is used for the URL of the user page etc. Only half-width alphanumeric characters and some symbols can be used."
```

Comment:
The URL of pages you create will contain your User ID. Your User ID can consist of letters, numbers, and some symbols.